clang/cmake/caches/Release.cmake

```diff
@@ -55,15 +55,23 @@ set(STAGE1_RUNTIMES "compiler-rt")
 
 if (LLVM_RELEASE_ENABLE_PGO)
   list(APPEND STAGE1_PROJECTS "lld")
-  set(CLANG_BOOTSTRAP_TARGETS
+  set(tmp_targets
     generate-profdata
     stage2-package
     stage2-clang
+    stage2
     stage2-install
     stage2-check-all
     stage2-check-llvm
     stage2-check-clang CACHE STRING "")
```

Comment:
@tstellar I don't know but I guess the `CACHE STRING ""` can be removed for the `tmp_targets`, right?